Makefile

```diff
@@ -2,7 +2,8 @@ SHELL := /bin/bash
 
 NS ?= abhinavsingh
 IMAGE_NAME ?= proxy.py
-VERSION ?= v$(shell python -m proxy --version)
+#VERSION ?= v$(shell bash -c "python -m setuptools_scm --version \| awk '{print$3}'")
```

Comment:
I think we want to use this one once upstream tag has been set?

I added a `v2.3.1` tag and now:

```
python -m setuptools_scm --version | awk '{print$3}' 
2.3.2.dev140+g094e30d
```

```
python -m setuptools_scm --version | awk '{print $$3}' | sed 's/\+/--/'
Version
```

Reply:
Aah, I noticed double `$$` used due to escape reasons.

- Any specific reason why you uses `--` and not `-`?  Within makefile the `VERSION` value currently resolves to `v2.3.2.dev140--g094e30d.d20211113`
- Also how do we go from `2.4` or `3.0`, may be answer is in `scmtools` flag?

Reply:
I guess I forgot to drop the commented-out line...

Reply:
> Aah, I noticed double `$$` used due to escape reasons.

Yeah, it took me a while to fight this problem. Make has syntax that is too complex and is not really necessary for Python projects. It'd be great to migrate to `tox` entirely but that's a separate topic.

> * Any specific reason why you uses `--` and not `-`?  Within makefile the `VERSION` value currently resolves to `v2.3.2.dev140--g094e30d.d20211113`

No reason, I just needed to replace that plus and thought that maybe a single dash may be ambiguous. FWIW I can replace it with one dash, it won't affect anything.

> * Also how do we go from `2.4` or `3.0`, may be answer is in `scmtools` flag?

By tagging the commit.